reframe/frontend/cli.py

```diff
@@ -410,7 +410,11 @@ def main():
     # Setup the check loader
     if options.checkpath:
         load_path = []
-        for d in options.checkpath:
+        if len(options.checkpath) > 1:
+            printer.warning("using command line option `--checkpath' "
+                            "multiple times is deprecated, please use "
```

Comment:
Again, I would suggest to warn the user that only the last check path will be taken into account, unless we keep supporting multiple `--checkpath`.

Reply:
Since we are moving to 3.0, just remove the warning.